src/AudioUnit/AUEnums.cs

```diff
@@ -187,7 +187,9 @@ public enum AudioObjectPropertySelector : uint
 		TranslateUIDToBox = 1969841250, // 'uidb'
 		ClockDeviceList = 1668049699, //'clk#'
 		TranslateUidToClockDevice = 1969841251, // 'uidc',
+		[Obsolete ("Use the 'ProcessIsMain' element instead.")]
```

Comment:
You are missing the deprecated attrs:

```
[Deprecated (PlatformName.TvOS, 15,0)]
```
you will have to add them for each of the supported platforms.

Reply:
unlike the other enum this one has a different value so it's useful to know when it should be avoided